tests/integration/go_ethereum/conftest.py

```diff
@@ -94,14 +94,17 @@ def base_geth_command_arguments(geth_binary, datadir):
         geth_binary,
         "--datadir",
         str(datadir),
-        "--nodiscover",
-        "--fakepow",
+        "--dev",
+        "--dev.period",
+        "5",  # dev.period > 1 for tests which require pending blocks
+        "--password",
+        os.path.join(datadir, "keystore", "pw.txt"),
     )
 
 
 @pytest.fixture(scope="module")
 def geth_zipfile_version(get_geth_version):
-    if get_geth_version.major == 1 and get_geth_version.minor in [10, 11]:
+    if get_geth_version.major == 1 and get_geth_version.minor in [10, 11, 12, 13]:
```

Comment:
nit: `py-geth` dropped support for 1.10.x a while ago, probably ok to do here too.